tests/test_integrate.py

```diff
@@ -143,47 +143,56 @@ def create_logline(ip, date, repo):
 
 @st.composite
 def log_data(draw):
-    ip_sample = st.lists(st.ip_addresses(), 10, unique=True)
+    ip_sample = st.lists(st.ip_addresses(), min_size=10, max_size=10, unique=True)
     repo = st.sampled_from(["Fedora", "epel-7", "centos8"])
     ips = draw(ip_sample)
     today = datetime.datetime.now()
     dates = [today - datetime.timedelta(days=d, hours=i) for i in range(1, 2) for d in range(1, 14)]
 
-    return list(
-        sorted(((date, ip, draw(repo)) for ip in ips for date in dates), key=lambda x: x[0])
-    )
+    return sorted(((date, ip, draw(repo)) for ip in ips for date in dates), key=lambda x: x[0])
 
 
-@settings(suppress_health_check=(HealthCheck.too_slow,))
-@given(log_data())
-@pytest.mark.skip(reason="Zuul doesn't like this")
+@given(loglines=log_data())
 def test_log(loglines):
     with tempfile.TemporaryDirectory() as tmp_dir:
+        rawdb = f"{tmp_dir}/test.db"
+        totalsdb = f"{tmp_dir}/test_generated_totals.db"
         matcher = CountmeMatcher
-        args = Args(
-            writer=make_writer("sqlite", str(tmp_dir + "/test.db"), matcher.itemtuple),
+        writer = make_writer("sqlite", rawdb, matcher.itemtuple)
+        matcher = matcher
+        dupcheck = True
+        index = True
+        header = True
+        progress = False
+        matchmode = "countme"
+        #        format = "csv"
+        #        logs = []
+        sqlite = rawdb
+        parse_from_iterator(
+            [(create_logline(ip, date, repo) for date, ip, repo in loglines)],
             matcher=matcher,
-            dupcheck=True,
-            index=True,
-            header=True,
-            progress=False,
-            matchmode="countme",
-            format="csv",
-            logs=[],
-            sqlite=str(tmp_dir + "/test.db"),
+            writer=writer,
+            dupcheck=dupcheck,
+            index=index,
+            matchmode=matchmode,
+            sqlite=sqlite,
+            header=header,
```

Comment:
Hmm, I’d put these directly into the call – `matcher` exists because `matcher.itemtuple` is needed.